src/picknik_ur_gazebo_config/objectives/force_exceeds_threshold.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root BTCPP_format="4" main_tree_to_execute="Force Exceeds Threshold">
+    <BehaviorTree ID="Force Exceeds Threshold" _description="Uses a Force Torque Sensor topic to check if input End Effector force exceeds threshold. This subtree is used to overwrite these values for each package." _favorite="true">
```

Comment:
I would not tag this as favorite

Reply:
I also removed the UpdateAdmittanceController and ReZeroFTS from favorites, as these should just be called in the objectives requiring them imo and are otherwise rarely used clutter. Lmk if this isn't the case though.